store/transport.go

```diff
@@ -0,0 +1,97 @@
+package store
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+
+	kithttp "github.com/go-kit/kit/transport/http"
+	"github.com/gorilla/mux"
+	"github.com/xmidt-org/argus/model"
+)
+
+// request URL path keys
+const (
+	bucketVarKey = "bucket"
+	idVarKey     = "id"
+)
+
+const (
+	bucketVarMissingMsg = "{bucket} URL path parameter missing"
+	idVarMissingMsg     = "{id} URL path parameter missing"
+)
+
+// Request and Response Headers
+const (
+	ItemOwnerHeaderKey  = "X-Xmidt-Owner"
```

Comment:
I don't think it belongs in this pr, but can we make these headers configurable?  If argus is used in a context outside of Xmidt, they seem out of place.